tests/x11/exiv2.pm

```diff
@@ -96,13 +96,15 @@ sub run {
     script_run "eog 20190201_154421.jpg", 0;
     assert_screen "exiv2_rename_test";
     send_key 'alt-f4';
+    wait_still_screen(1);
```

Comment:
Unrelated change?

Reply:
This is failure which were blocking VR, it has separate commit.
I didn't see any poo so i just added it to this PR.